spec/features/breadcrumbs_spec.rb

```diff
@@ -50,29 +50,31 @@
 
     scenario 'user navigates search, prosecution case and defendant details pages', :vcr do
       when_i_choose_search_filter 'A case by URN'
-      when_i_search_for 'MOGUERBXIZ'
+      when_i_search_for 'MVIFVOIPYU'
 
-      click_link('MOGUERBXIZ', match: :first)
-      expect(page).to have_current_path(prosecution_case_path('MOGUERBXIZ'))
+      click_link('MVIFVOIPYU', match: :first)
+      expect(page).to have_current_path(prosecution_case_path('MVIFVOIPYU'))
 
-      then_has_case_details_breadcrumbs('MOGUERBXIZ')
+      then_has_case_details_breadcrumbs('MVIFVOIPYU')
 
-      click_link('Josefa Franecki')
+      click_link('Lawerence Predovic Ortiz')
       expect(page).to have_current_path(%r{/laa_references/.+})
 
-      then_has_defendant_details_breadcrumbs('MOGUERBXIZ', 'Josefa Franecki')
+      then_has_defendant_details_breadcrumbs('MVIFVOIPYU', 'Lawerence Predovic Ortiz')
 
-      click_breadcrumb 'Case MOGUERBXIZ'
+      # TODO: I have (temporarily) removed the case breadcrumb as this was
+      #       causing issues.
+      # click_breadcrumb 'Case MVIFVOIPYU'
 
-      expect(page).to have_current_path(prosecution_case_path('MOGUERBXIZ'))
-      then_has_case_details_breadcrumbs('MOGUERBXIZ')
+      # expect(page).to have_current_path(prosecution_case_path('MVIFVOIPYU'))
+      # then_has_case_details_breadcrumbs('MVIFVOIPYU')
```

Comment:
do you need help with getting the breadcrumb in?

Reply:
👍 done - many thanks for your assistance with getting the case breadcrumb working